manifests/server.pp

```diff
@@ -103,6 +104,7 @@
   $service_provider                                = $postgresql::params::service_provider,
   $service_reload                                  = $postgresql::params::service_reload,
   $service_status                                  = $postgresql::params::service_status,
+  Enum[present, absent] $systemd_drop_in_ensure    = $postgresql::params::systemd_drop_in_ensure,
```

Comment:
to make the puppet-strings documentation more beautiful I suggest to add the data here, and not in params.pp:
```suggestion
  Enum[present, absent] $systemd_drop_in_ensure    = 'present',
```

Reply:
Yes, would break the existing style. In general, this is a topic on its own and should be done for the whole class/module in my opinion

Reply:
I'm not sure what's the Puppet rule here (if there is one); at Vox Pupuli we enforce new style guide (such as not using params.pp) before modules are converted.

Reply:
quoted the present.